linker/shared/src/main/scala/org/scalajs/linker/standard/CoreSpec.scala

```diff
@@ -60,13 +62,16 @@ private[linker] object CoreSpec {
     new CoreSpec(
         semantics = Semantics.Defaults,
         moduleKind = ModuleKind.NoModule,
-        esFeatures = ESFeatures.Defaults)
+        esFeatures = ESFeatures.Defaults,
+        linkTimeProperties = new LinkTimeProperties(Semantics.Defaults,
+          ESFeatures.Defaults))
```

Comment:
As is, `linkTimeProperties` can entirely be derived from the other fields of `CoreSpec`, so it can be a `val` in the body of `CoreSpec`.

We should pass as parameters custom link-time properties, if/when we add those.